CODINGSTYLE.md

```diff
@@ -43,6 +43,34 @@ A continuation line is a line appearing because we broke something that should h
 Typically, this means inside parentheses (formal or actual parameters of methods and constructors), and a long `extends` clause.
 
 Note that breaking a line right after the `=` sign of an initialization or assignment is *not* considered a continuation line, because it's not really breaking the line: instead, we just choose to put the rhs on its dedicated line, indented 2 spaces (similarly to the branches of an `if`).
+For example:
+
+```scala
+val x =
+  aLongFunctionCall()
+```
+
+Further, parenthesized lists that have a single element per line are not considered continuation lines.
+For example, the following two are allowed:
+
+```scala
+// "Binpacked style"
+f(arg1, arg2,
+    arg3, arg4)
+
+// "List style"
+f(
+  arg1,
+  arg2,
+  arg3,
+  arg4
+)
+```
+
+Notes about the list style:
+* The parenthesis must be on individual lines.
+* A trailing comma will become mandatory if/once we drop 2.11.
+* This style is relatively new, so a lot of code does not comply to it; apply the boy scout rule where feasible.
```

Comment:
Perhaps add
```suggestion
* This style is relatively new, so a lot of code does not comply to it; apply the boy scout rule where feasible, as long as it does not gratuitously affect `git blame`.
```
? Or some better formulation?

We should just avoid reindenting a dozen-line list when we only change one of the lines, IMO (or worse, don't even change that list at all, but just stuff around).

Reply:
PTAL